src/main/java/seedu/moneygowhere/commands/ConsoleCommandDeleteRemarks.java

```diff
@@ -0,0 +1,17 @@
+package seedu.moneygowhere.commands;
+
+/**
+ * Stores the delete-expense command and its arguments.
+ */
+@SuppressWarnings("FieldMayBeFinal")
+public class ConsoleCommandDeleteRemarks extends ConsoleCommand {
+    private int expenseIndex;
+
+    public ConsoleCommandDeleteRemarks(int expenseIndex) {
+        this.expenseIndex = expenseIndex;
+    }
+
+    public int getExpenseIndex() {
+        return expenseIndex;
+    }
+}
```

Comment:
May I ask what is the rationale behind the Delete-Remark command? I believe using the command Edit-Expense -e 0 -r "" would remove the remark as well

Reply:
I thought that one of the user stories explicitly stated that you should be able to delete-remarks.

Reply:
I see. Do you think its better to have an explicit command to delete remarks or would using the existing Edit-Expense command be better?

Reply:
You make a fair point though, given our target audience, I think it would be better to stick with the existing Edit-Expense command.

Reply:
I'll remove the command entirely!

Reply:
Thanks!